Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -101,8 +107,22 @@ void ProcessThrottler::invalidateAllActivities()
     PROCESSTHROTTLER_RELEASE_LOG("invalidateAllActivities: END");
 }
 
+void ProcessThrottler::holdTemporaryForegroundAssertion()
+{
+    m_shouldHoldTemporaryForegroundAssertion = true;
```

Comment:
Why a new boolean? Why can't we just hold a ProcessActivity in a data member like we usually do?
Maybe we'd hold it from 8 minutes after no longer being visible?